Libraries/VaahPayPal.md

```diff
@@ -0,0 +1,77 @@
+# VaahPaypal
+> Small Laravel Helpers
+
+### Installation
+
+VaahPaypal provides to you a simple way to integrate Paypal in your Laravel application.
+
+### Dependencies
+- [PayPal-PHP-SDK](https://github.com/paypal/PayPal-PHP-SDK)
+
+### Installation
+
+```shell script
+composer require paypal/rest-api-sdk-php
+```
+
+[comment]: <> (Add Facade in `config/app.php`:)
+
+[comment]: <> (```php)
+
+[comment]: <> ('aliases' => [)
+
+[comment]: <> (...)
+
+[comment]: <> ('VaahStripe' => WebReinvent\VaahExtend\Facades\VaahStripe::class,)
+
+[comment]: <> (...)
+
+[comment]: <> (])
+
+[comment]: <> (```)
+
+Add env configuration:
+```env
+PAYPAL_MODE=sandbox
+PAYPAL_SANDBOX_CLIENT_ID=xxxxxxxxxxxxxxx
```

Comment:
@we-himanshu-r001-v1 write where and how to find `PAYPAL_SANDBOX_CLIENT_ID` and `PAYPAL_SANDBOX_CLIENT_SECRET`